Gemfile.lock

```diff
@@ -19,13 +19,13 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (4.1.14.1)
-      actionview (= 4.1.14.1)
-      activesupport (= 4.1.14.1)
+    actionpack (4.1.15)
```

Comment:
This is a lot of updates, did you run the integration tests against a PayPal sandbox (`rake test:remote:spec`)? I'm a bit nervous about side effects.

Alternatively, we could tackle these upgrades separately if we need/want them.